vercel/examples
# Update for Feature Flag Examples

### Update for Feature Flag Apple Store Example

- This PR updates the Apple Store example to to the latest API using middleware and edge functions.
- Additionally, this PR adds more functionality to showcase the power of doing Feature Flag at the Edge.
- Updates ENV vars to match the ones provided by Upstash.

### Demo URL

Preview URL

## How to Test
- Go to the index, check the store
- Close the store going to `/api/store/close`
- Check the index
- Open the store again `/api/store/open`
- DONE! 

### Type of Change

- [X] Example updates (Bug fixes, new features, etc.)
- [ ] Other (changes to the codebase, but not to examples)
